make/acceptance-tests.mk

```
@@ -72,11 +72,13 @@ setup-venv: ## Setup virtual environment
 
 .PHONY: test-acceptance-setup
 test-acceptance-setup: setup-venv ## Setup the environment for the acceptance tests
+ifeq ($(CLUSTER_PROVIDER), kind)
+	@(kind get clusters | grep primaza | xargs -I@ kind delete cluster --name @) || true
```

this can conflict with primazactl's ones, I'd propose to grep `primaza-`

```suggestion
	@(kind get clusters | grep 'primaza-' | xargs -I@ kind delete cluster --name @) || true
```